### PR TITLE
Clarify grpcReply argument in grpc client factory

### PR DESCRIPTION
**Pull request by jupp0r**

### CODE REVIEW

File: transport/grpc/client.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ type Client struct {
 }
 
 // NewClient constructs a usable Client for a single remote endpoint.
+// Pass an uninitialized protobuf message of the rpc response type as
+// the grpcReply argument.
```
Comment (peterbourgon) [inline transport/grpc/client.go]: - s/rpc/RPC/
- s/protobuf/Protobuf/
- s/uninitialized/zero-value/

**Comment by peterbourgon** [comment]
After that, LGTM!

**Comment by jupp0r** [comment]
Done

**Comment by peterbourgon** [comment]
Danke!

**Pull request merged**